Neos.Http.Factories/composer.json

```diff
@@ -9,7 +9,7 @@
     "require": {
         "php": "^7.1",
         "psr/http-factory": "^1.0",
-        "guzzlehttp/psr7": "^1.4, !=1.8.0 || ~2.0"
+        "guzzlehttp/psr7": "^1.4.1, !=1.8.0 || ~2.0"
```

Comment:
Required, because 1.4.0 throws an exception when `withPath()` is called without a leading slash - we need to fix this in Flow - see #2473